app/ui-react/packages/ui/src/Customization/apiClientConnectors/create/ApiClientConnectorCreateSecurity.tsx

```diff
@@ -105,11 +103,11 @@ export const ApiClientConnectorCreateSecurity: React.FunctionComponent<IApiClien
     setValid(isValid(selectedType, authUrl, newUrl));
   };
   return (
-    <Card style={{ maxWidth: '600px', margin: ' auto' }}>
-      <CardHeader>
+    <Stack style={{ maxWidth: '600px', margin: ' auto' }} gutter="md">
```

Comment:
Yeah, maybe lose the `margin` here and let it right-justify, that seems to be the norm in the pf4 wizard.